app/lib/index.js

```diff
@@ -419,7 +419,6 @@ app.createEditorWindow = function() {
   const windowOptions = {
     resizable: true,
     show: false,
-    title: 'Camunda Modeler' + getTitleSuffix(app.metadata.version),
```

Comment:
I personally found this useful, why not keep it?

![image](https://github.com/camunda/camunda-modeler/assets/58601/7bd8d023-180c-4809-9d78-2b27df18ef2d)

Reply:
It's a duplicate of 
![image](https://github.com/camunda/camunda-modeler/assets/28307541/a955640b-594f-4a1b-97d6-053075ea2e97)

Reply:
If we keep the code above, we set the title in two places:

1) main process
2) HTML

But on MacOS the HTML part takes precedence.

Reply:
Long term, we could consider adding context to the title, e.g. the current tab. But it's outside of the scope of the linked issue.

Reply:
OK, let's decide here and now that we ditch `dev` tag, and if we ever add it (again), then we do it inside the app.